src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts

```diff
@@ -0,0 +1,134 @@
+import { html, render } from "lit-html";
+import { IIConnection } from "../../../utils/iiConnection";
+import { renderManage } from "../../manage";
+import { withLoader } from "../../../components/loader";
+import { verifyDevice } from "./verifyTentativeDevice";
+import {
+  Countdown,
+  formatRemainingTime,
+  setupCountdown,
+} from "../../../utils/countdown";
+import {
+  DeviceData,
+  IdentityAnchorInfo,
+} from "../../../../generated/internet_identity_types";
+
+const pageContent = (userNumber: bigint, endTimestamp: bigint) => html`
+  <div class="container">
+    <h1>Add New Remote Device</h1>
+    <p>
+      Device registration mode enabled for Identity Anchor
+      <b>${userNumber}</b>. Please follow these steps to add your new device:
+    </p>
+    <ol class="instruction-steps">
+      <li>
+        On your <i>new</i> device:<br />
```

Comment:
nit: there's a lot of `<i>` here and in other files. I think `<em>` is more appropriate since you really want to _emphasize_. Same goes with `<b>` (and `<strong>`).